examples/pytorch/seal/gcn_layers.py

```diff
@@ -0,0 +1,517 @@
+# Most codes from dgl.nn.pytorch
+# Adjust codes in forward for utilizing edge weight in message passing.
```

Comment:
The `SAGEConv` and `GraphConv` module has already supported `edge_weight`.

Reply:
Ok